features/siddhi-tryit/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+~ Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@keizer619 
Fixed in commit https://github.com/wso2/carbon-event-processing/pull/179